modules/v2x/common/v2x_proxy_gflags.cc

```diff
@@ -36,6 +40,19 @@ DEFINE_int64(v2x_carstatus_timer_frequency, 10,
 DEFINE_string(hdmap_file_name,
```

Comment:
map file name as a gflag is already defined: https://github.com/ApolloAuto/apollo/blob/master/modules/common/configs/config_gflags.h